File: teste.lua
function primo(num)
	if num < 2 then
		return false
	end
	for i = 2,(num-1) do
		if num % i == 0 then
			return false
		end
	end
	return true
end

for i=0,100000 do
	if primo(i) then
		print(i)
	end
end

File: teste.rs
fn primo(num:i32) -> bool {
	if num < 2 {
		return false;
	}
	for i in 2..num {
		if num % i == 0 {
			return false;
		}
	}
	return true;
}

fn main(){
	for i in 0..100000 {
		if primo(i) {
			println!("{}", i);
		}
	}	
}
